src/BabelPluginTypes.ts

```diff
@@ -6,7 +6,11 @@ export interface BabelOptions {
   filename: string;
 }
 export interface ParseOptions {
-  plugins: Array<string>;
+  sourceType?: 'module' | 'script';
```

Comment:
It seems like `'script'` isn't used as a value anywhere? Or is that user-configurable / future-proofing?

Reply:
Ideally all these things would be provided by `.d.ts` files that come with e.g. `@babel/parser`, but we don't currently live in that world and I'm not sure how to pull them from DefinitelyTyped when the packages are namespaced, so I just manually write out a subset here. These are the two allowed values for `sourceType`, so I just encoded that here.